### ISSUE TITLE
Feature: [Locate SDKs] Ability/option to run a search of your system for existing SDKs

### ISSUE THREAD

**Issue by MostHated** [sdkman/sdkman-cli#1044]
- [ ] I have read the [CONTRIBUTING guidelines](CONTRIBUTING.md)

**Feature request**

Hey there,
 Love the tool so far. It would be great if there was an option to search/detect existing SDK's already, either via common installation paths, or env vars (ex. go env / $GOROOT,  dotnet --info / $DOTNET_ROOT, etc) as I can sometimes end up with them all over the place, or simply not remembering where one was that I installed last year (probably much more) and never bothered to update, yet I have the new version installed in my new "consolidated location" (which I try to remember to install things to, but don't always). So there it sits, outdated, forgotten, hoping to be located.

Keep up the great work!
Thanks,
-MH

**Comment by marc0der** [comment]
Hi @MostHated, glad you like it so far!

I'm not sure what you mean by "search/detect existing SDKs". All the SDKs installed by SDKMAN live in one place under the `~/.sdkman/candidates` folder. Besides that, you have no real need to go poking in there as everything you need to manage your SDKs (including the setting of env vars) is managed by SDKMAN. The idea is that you don't need to do any of this by hand anymore, but delegate all of this "management stuff" to our tool.

For full instructions on how to use SDKMAN, please read the [usage page](https://sdkman.io/usage) on our website. If still in doubt, you can reach us on our [community slack](https://slack.sdkman.io).

**Comment by MostHated** [comment]
Yeah, I definitely understand that, I was referring to adding existing SDK's to a new sdkman install. I just installed it and thought it would be quite nice if it could search for and use the many SDK's I currently have installed and keep them updated. If not possible, I can definitely understand, was just an idea.
